sql/mob_groups.sql

```diff
@@ -24,25 +24,23 @@ DROP TABLE IF EXISTS `mob_groups`;
 /*!40101 SET character_set_client = utf8 */;
 CREATE TABLE `mob_groups` (
   `groupid` int(10) unsigned NOT NULL,
-  `poolid` int(10) unsigned NOT NULL DEFAULT 0,
-  `zoneid` smallint(3) unsigned NOT NULL DEFAULT 0,
-  `name` varchar(24) DEFAULT NULL,
```

Comment:
Looks like you goofed when merging the masters changes into your branch, as the diff indicates undoing the changes rather than integrating them.

Reply:
nvm I'm looking at an old diff. Thanks github!

Reply:
No, you're right, I dun goofed. I've messed up `mob_groups` and `mob_spawn_points`

Reply:
explorer moogle stuff to?